src/env-inl.h

```diff
@@ -916,6 +916,34 @@ inline void Environment::RemoveHeapSnapshotNearHeapLimitCallback(
                                         heap_limit);
 }
 
+inline v8::CpuProfilingResult Environment::StartCpuProfile(
+    v8::Local<v8::String> title) {
+  if (!cpu_profiler_) {
+    cpu_profiler_ = v8::CpuProfiler::New(isolate_);
+  }
+  v8::CpuProfilingResult result = cpu_profiler_->Start(title, true);
+  if (result.status == v8::CpuProfilingStatus::kStarted) {
+    node::Utf8Value key(isolate_, title);
+    pending_profiles_.insert({*key, result.id});
+  }
+  return result;
+}
+
+inline v8::CpuProfile* Environment::StopCpuProfile(
+    v8::Local<v8::String> title) {
+  if (!cpu_profiler_) {
+    return nullptr;
+  }
+  node::Utf8Value key(isolate_, title);
+  auto it = pending_profiles_.find(*key);
```

Comment:
```suggestion
  auto it = pending_profiles_.find(key.ToStringView());
```

Reply:
It seem `find` does not receive`string_view` type.